### DISCUSSION TITLE
Welcome to async-firebase Discussions!

### DISCUSSION THREAD

**Discussion by akalex** [healthjoy/async-firebase#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eighthcolor22** [comment]
Hi!

We're using a fork of async-firebase in our project. The primary reason for using the fork is the functionality to subscribe and unsubscribe devices to topics.

Do you have any plans to add this functionality to the official async-firebase library? We'd be happy to contribute a pull request with this feature.

Thanks.

**Reply by akalex** [reply]
Hi there!

Thanks for bringing this up. Extending the library with the ability to subscribe to/unsubscribe from a topic is a great idea. That was something other than what was on our to-do list for the short term. Therefore, I would be happy if you could contribute.
